chrome/content/zotero/itemPane.xul

```diff
@@ -33,13 +33,23 @@
 	
 	<vbox id="zotero-item-pane" zotero-persist="width">
 		<!-- Trash -->
-		<hbox id="zotero-item-top-buttons-holder" hidden="true">
+		<hbox id="zotero-item-pane-top-buttons-trash" hidden="true">
```

Comment:
Need to update this id in itemPane.css. Also, the style of the new buttons should match the styles used for the trash buttons there (e.g., `-moz-appearance: toolbar;`, and `-moz-box-flex` on the buttons), so should use a class for both.